### ISSUE TITLE
Overlap of reporting status tags with indicator names on goal pages

### ISSUE THREAD

**Issue by LucyGwilliamAdmin** [open-sdg/open-sdg#1195]
**Please complete the following information** 
Site repository URL: All
Open SDG version: At least latest - not sure about other goal layouts

**Describe the bug**
On vertical by target goal pages, the reporting status tag overlaps with the indicator name/link when zoomed into certain levels.

**To Reproduce**
1. Go to https://sdgdata.gov.uk/1/
2. Zoom into 200%
3. Scroll down to 1.4.1
Note: if the error doesn't display, try with different zoom levels and check different indicators

**Expected behavior**
No overlap between reporting status tag and indicator name

**Screenshots**
![image](https://user-images.githubusercontent.com/43404358/117296207-b1587f00-ae6c-11eb-821c-a2e846221268.png)

**Additional context**
From accessibility report:
Ensure that when users increase the size of pages to 400% and the browser width 1280px to view the page content in one column, they can do this without having to scroll in two directions, and without the loss of content or functionality.For more information, please refer to [F102: Failure of Success Criterion 1.4.10 due to content disappearing and not being available when content has reflowed](https://www.w3.org/WAI/WCAG21/Techniques/failures/F102).

**Comment by brockfanning** [comment]
@LucyGwilliamAdmin I can't recreate this, after testing on Firefox, Edge, and Chrome. What browser was used to make that screenshot? Or is it possible there is some other factor involved, like maybe increased font-size at the OS-level?

**Comment by LucyGwilliamAdmin** [comment]
@brockfanning I'm using Chrome and don't have font size increased at OS level. What about if you decrease window width? I haven't needed to do that but might help with recreation?

**Comment by brockfanning** [comment]
@LucyGwilliamAdmin I found it - the issue was only happening with "Exploring data sources", because it's longer than the others. I'm trying an approach of giving that tag a max width, so that longer tags will wrap.
